### ISSUE TITLE
Publish A11y - Design table

### ISSUE THREAD

**Issue by michaelcattell** [dwp/design-system#295]

**Comment by michaelcattell** [comment]
A quick mock up to help start the conversation into what we need to include etc.
The first paragraph to be a summary of the findings, the table below includes links to the excel doc or a separate page that is the html version of excel (for accessibility reasons)?
The next column (WGAG compilant) could just be a simple yes or no with titles? Or should this bit be nothing OR an explanation of what test is what next to the link?

![a11y--table](https://user-images.githubusercontent.com/52162960/143069520-17c07763-a87c-4db8-8f51-790eda7e2f74.jpg)

**Comment by michaelcattell** [comment]
After discussing with the team, leaning heavily on carbon design for inspiration (with demo data):
![image](https://user-images.githubusercontent.com/52162960/144066862-47815fb6-c08b-4371-895f-f443f67fe002.png)

Things we could/should include:
Date of test, things we know (we know it doesnt work with dragon), future fixes/plans

Things worth noting: https://design-system.service.gov.uk/components/accordion/#known-issues-and-gaps

**Comment by michaelcattell** [comment]
After a discussion with Jon H, we decided on two layouts to trial and send onto Craig A for feedback. 
4 mock ups , similar layouts but showing different scenarios (all passed + one failed):
**Additional info:**
This is going to be on reviewed patterns (hopefully all)
We are going to turn the excel doc into html, so the link will not surface the excel doc
The data included is just demo data, one mock up tries to educate to the users what JAWS is, one is minimal data with a link to github issue where we’ll put the report on there

(also added a github discussion link for use: https://github.com/dwp/design-system-community-backlog/discussions/82)

![dac--more--no](https://user-images.githubusercontent.com/52162960/144414517-6bfb7ff5-3a2f-4d1d-8715-1583f8d2eea6.jpg)
![dac--more--yes](https://user-images.githubusercontent.com/52162960/144414522-041d3700-290e-4231-9d8c-0eb9742eb45c.jpg)
![dac--less--no](https://user-images.githubusercontent.com/52162960/144414493-d79e5ec4-8a98-47b1-b4b7-5db37d5d1bed.jpg)
![dac--less--yes](https://user-images.githubusercontent.com/52162960/144414508-33315243-7476-4ea1-b6dd-9a5e4616d26c.jpg)

**Comment by michaelcattell** [comment]
Recd feedback from Craig:

![image](https://user-images.githubusercontent.com/52162960/144882059-04b4cc01-d462-4877-bee5-1c1d538863cc.png)

**Comment by michaelcattell** [comment]
With the above feedback and chats with the team, we discussed our concern around the amount of work to export a page where the results have been outputted to (html page instead of an excel doc), so initially we would let the user "get in touch" to request the form.

![image](https://user-images.githubusercontent.com/52162960/144883498-ea6ea8cd-e9fb-4d6a-b7c9-2ea1d8221d76.png)

**Comment by jonhurrell** [comment]
I'm happy with this as a short to medium term tactic to give assurance. Long term will be to give visibility of the DAC tests and result of each component in HTML. @michaelcattell During the pattern hour we could ask for some feedback to see people think this meets their needs?

**Comment by martinwake** [comment]
Left comments on @michaelcattell 's Mural board

**Comment by jonhurrell** [comment]
Closing, references in https://github.com/dwp/design-system/issues/171
